### PR TITLE
Update `psr/log` version

### PR DESCRIPTION
**Pull request by rustamwin**
| Q             | A
| ------------- | ---
| Is bugfix?    | ✔️/❌
| New feature?  | ✔️/❌
| Breaks BC?    | ✔️/❌
| Fixed issues  | comma-separated list of tickets # fixed by the PR, if any

### CODE REVIEW

File: tests/Log/SimpleLoggerTest.php [hunk 1]
```diff
@@ -146,6 +150,9 @@ public function testPsrLogInterfaceMethods($message, string $expected): void
 
     public function testLogWithResourceMessage(): void
```
Comment (arogachev) [inline tests/Log/SimpleLoggerTest.php]:
```suggestion
    /**
     * @requires PHP < 8.0
     */
    public function testLogWithResourceMessage(): void
```

Comment (arogachev) [reply tests/Log/SimpleLoggerTest.php]: https://phpunit.readthedocs.io/en/9.5/incomplete-and-skipped-tests.html#incomplete-and-skipped-tests-skipping-tests-examples-databaseclassskippingtest-php

File: tests/Log/SimpleLoggerTest.php [hunk 2]
```diff
@@ -113,6 +113,10 @@ public function __toString(): string
      */
     public function testPsrLogInterfaceMethods($message, string $expected): void
```
Comment (arogachev) [inline tests/Log/SimpleLoggerTest.php]:
```suggestion
    /**
     * @requires PHP < 8.0
     */
    public function testPsrLogInterfaceMethods($message, string $expected): void
```

Comment (arogachev) [reply tests/Log/SimpleLoggerTest.php]: https://phpunit.readthedocs.io/en/9.5/incomplete-and-skipped-tests.html#incomplete-and-skipped-tests-skipping-tests-examples-databaseclassskippingtest-php

**Review by arogachev — CHANGES_REQUESTED** [review]
Could be simplified a bit (see suggestions).

**Review by arogachev — APPROVED** [review]

File: composer.json [hunk 3]
```diff
@@ -20,7 +20,7 @@
         "php": "^7.4|^8.0",
         "psr/container": "^1.0|^2.0",
         "psr/event-dispatcher": "^1.0",
-        "psr/log": "^1.1",
+        "psr/log": "^1.0|^2.0|^3.0",
```
Comment (roxblnfk) [inline composer.json]:
```suggestion
        "psr/log": "^1.1|^2.0|^3.0",
```

File: tests/Log/SimpleLoggerTest.php [hunk 4]
```diff
@@ -110,6 +110,8 @@ public function __toString(): string
      *
      * @param mixed $message
      * @param string $expected
+     *
+     * @requires PHP < 8.0
```
Comment (roxblnfk) [inline tests/Log/SimpleLoggerTest.php]: What is this for?

File: tests/Log/SimpleLoggerTest.php [hunk 5]
```diff
@@ -144,6 +146,9 @@ public function testPsrLogInterfaceMethods($message, string $expected): void
         $this->assertSame($expected, $messages[8]['message']);
     }
 
+    /**
+     * @requires PHP < 8.0
+     */
```
Comment (roxblnfk) [inline tests/Log/SimpleLoggerTest.php]: What is this for?

File: tests/Log/SimpleLoggerTest.php (hunk 4 above)
Comment (rustamwin) [reply tests/Log/SimpleLoggerTest.php]: For `psr/log` ^1.1 version

File: tests/Log/SimpleLoggerTest.php (hunk 5 above)
Comment (rustamwin) [reply tests/Log/SimpleLoggerTest.php]: For `psr/log` ^1.1 version

File: tests/Log/SimpleLoggerTest.php (hunk 4 above)
Comment (roxblnfk) [reply tests/Log/SimpleLoggerTest.php]: Are you sure these tests are needed?

Comment (rustamwin) [reply tests/Log/SimpleLoggerTest.php]: Yes.

**Review by samdark — APPROVED** [review]

**Pull request merged**